docs/guides/integration-google-ss.md

```diff
@@ -0,0 +1,98 @@
+
+When an encrypted signal is cached, the secure signals feature does not execute the handler to generate a new signal. Because of this, it is necessary to clear the cache before and after data capture.
+
+Since the secure signals feature does not provide a way to delete or invalidate a specific ID, publishers must call the `window.googletag.secureSignalProviders.clearAllCache()` function to clear all shared encrypted signals as part of their data capture workflows.
+
+The following is an example of calling the `window.googletag.secureSignalProviders.clearAllCache()` function:
+
+```
+window.googletag = window.googletag || { cmd: [] };
+window.googletag.cmd.push(function () {
+  window.googletag.secureSignalProviders =
+    window.googletag.secureSignalProviders || [];
+  window.googletag.secureSignalProviders.clearAllCache();
+});
+```
+
+### Server-Side Integration
+
+So that it can share encrypted signals, the hosted auto-loaded secure signals script must be able to make an asynchronous call to the `window.getEuidAdvertisingToken` function and, in response, receive `advertising_token` as a string.
+
+It's important to make sure that the identity token is fresh. For a server-side integration, we recommend making a call to the [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint to get a fresh [advertising token](../endpoints/post-token-refresh.md#decrypted-json-response-format) from the JSON response.
+
+The following code is an example of how you could do this.
+
+```
+window.getEuidAdvertisingToken = async () => {
+  // Make a call to get a fresh identity token which could last for at least 12 hours.
+  const identity = await getFreshIdentity()
+  return JSON.parse(decodeURIComponent(identity)).advertising_token
+}
+```
+
+For details, see [Publisher Integration Guide, Server-Side](integration-publisher-server-side.md).
+
+A sample application is also available for server-side integration. See [Sample Applications](#sample-applications).
+
+### SDK for JavaScript Client-Side Integration
+
+If you're using the SDK for JavaScript version 3.4.0 or later, the EUID secure signals script uses the `getAdvertisingTokenAsync` function provided in the SDK to get the fresh advertising token, and then pushes the token to Google Ad Manager.
+
+<!-- (UID2/EUID diff re JS SDK ver at 20240813 UID2 3.0.0 EUID 3.4.0. 170) -->
+
+This script is hosted on CDN, and GPT automatically loads it with the secure signals feature. 
+
+For details, see [Client-Side Integration Guide for JavaScript](integration-javascript-client-side.md).
+
+A sample application is also available for integration using the SDK for JavaScript. See [Sample Applications](#sample-applications).
+
+## Sample Applications
+
+The following sample applications are available to illustrate how to integrate with the Google Ad Manager secure signals feature:
+
+- Server-Side EUID Integration Example:
+  - [Sample application](https://secure-signals-srvonly-integ.uidapi.com/)
```

Comment:
i really don't think we should show these UID2 examples until we have EUID specific example ready - it just confuses people i think. OR in the description here, you explicitly states _these examples are for UID2 but to use it for EUID you would need to do XYZ_....

Reply:
Commented out that section and references to it.